server/balancer.go

```diff
@@ -603,6 +610,8 @@ func (h *balanceHotRegionScheduler) balanceByPeer(cluster *clusterInfo) (*Region
 	return nil, nil, nil
 }
 
+// selectDestStoreByPeer selects a target store hold the region of the source region.
```

Comment:
-> selectDestStoreByPeer selects a target store to hold the region of the source region.